lib/messenger_bot.js

```diff
@@ -557,18 +554,27 @@ class MessengerBot extends BaseBot {
    * see botmaster's BaseBot #getUserInfo
    *
    * @param {string} userId id of the user whose information is requested
+   * @param {object} requestOptions
    */
-  __getUserInfo(userId) {
+  __getUserInfo(userId, requestOptions) {
     const options = {
       method: 'GET',
       uri: `${baseURL}/${userId}`,
-      qs: { access_token: this.credentials.pageToken },
+      qs: { access_token: this.__getAccessToken(requestOptions.sendAsPageId) },
       json: true,
     };
 
     return request(options);
   }
 
+  __getAccessToken(pageId = this.__associatedUpdate.recipient.id) {
```

Comment:
You won't be able to use default params if you want your tests to pass on node 4 and 5